src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -110,7 +110,7 @@ def test_source_waveform_parallel_broadcast(multi_instrument_session):
         timeout=5)
 
     pass_fail = multi_instrument_session.get_site_pass_fail(site_list='')
```

Comment:
Use something like the snippet below when this is made rep cap.
```
    pass_fail = multi_instrument_session.sites[0,3].get_site_pass_fail()
```

Reply:
Again, which ever PR gets merged last, will do it.